README.md

```diff
@@ -63,44 +63,122 @@ Please use a dedicated extension like [blamer-vs](https://marketplace.visualstud
 ## Settings
 Here is a table of settings with their default values. To change any of these, add the relevant Config key and value to your VSCode settings.json file.
```

Comment:
This will need re-wording

Reply:
Yes

Reply:
Suggestion?

Reply:
Here are all of the extension settings with their default values. To change any of these, add the relevant Config key and value to your VSCode settings.json file. Alternatively search for the config key in the settings UI.